documentation/3.1.0/dockerized/MentorEd-Setup-README.md

```diff
@@ -18,44 +18,62 @@ Once these prerequisites are in place, you're all set to get started with settin
 
     > Example Command: `mkdir mentoring && cd mentoring/`
 
-2.  **Download Docker Compose File:** Retrieve the **[docker-compose-mentoring.yml](https://github.com/ELEVATE-Project/mentoring/blob/master/src/scripts/setup/docker-compose-mentoring.yml)** file from the Mentoring repository and save it to the mentoring directory.
-
+    >**Caution:** Before proceeding, please ensure that the ports given here are available and open. It is essential to verify their availability prior to moving forward. You can run below command in your teminal to check this
     ```
-    curl -OJL https://github.com/ELEVATE-Project/mentoring/raw/master/documentation/3.1.0/dockerized/docker-compose-mentoring.yml
+    for port in 3000 3001 3002 6000 5001 4000 9092 5432 7007 2181 2707 3569; do
+    if lsof -iTCP:$port -sTCP:LISTEN &>/dev/null; then
```

Comment:
@sumanvpacewisdom are we checking this in composer file or its manual steps to check this ?

Reply:
its manual step , we need to copy and paste it @rakeshSgr